ui-admin/src/study/participants/participantList/ParticipantList.tsx

```diff
@@ -93,6 +103,9 @@ function ParticipantList({ studyEnvContext }: {studyEnvContext: StudyEnvContextT
     meta: {
       columnType: 'instant'
     },
+    // if we don't use an accessorFn, tanstack will spam the logs with
+    // 'undefined' warnings
+    accessorFn: row => row.participantUser?.lastLogin,
```

Comment:
if you're providing an accessorFn, I think you should remove the `accessorKey` definition